DEVX-2044: Document ccloud-stack option to reuse existing environment (#776)
confluentinc/examples: ccloud/docs/ccloud-stack.rst

Create a ccloud-stack
---------------------

> #. By default, the ``cloud-stack`` utility creates resources in the cloud provider ``aws`` in region ``us-west-2``. If this is the target provider and region, then create the stack by calling the bash script :devx-examples:`ccloud_stack_create.sh|ccloud/ccloud-stack/ccloud_stack_create.sh`. For additional options when configuring your ``ccloud-stack``, see :ref:`ccloud-stack-options`.

Reviewer: ```suggestion
#. By default, the ``cloud-stack`` utility creates resources in the cloud provider ``aws`` in region ``us-west-2``. If this is the target provider and region, create the stack by calling the bash script :devx-examples:`ccloud_stack_create.sh|ccloud/ccloud-stack/ccloud_stack_create.sh`. For more options when configuring your ``ccloud-stack``, see :ref:`ccloud-stack-options`.
```

   .. code:: bash

      ./ccloud_stack_create.sh

> #. In addition to creating all the resources in |ccloud| with associated service account and ACLs, running ``ccloud-stack`` also generates a local configuration file with all above connection information, useful for other demos/automation. View this file at ``stack-configs/java-service-account-<SERVICE_ACCOUNT_ID>.config``. It resembles:

Reviewer: @ybyzek For "with all above connection information," how about "with all the previous connection information" or "with all the earlier connection information?"

Author: Refined a bit

      ./ccloud_stack_destroy.sh stack-configs/java-service-account-<SERVICE_ACCOUNT_ID>.config

.. _ccloud-stack-options:

================
Advanced Options
================

> #. By default, the ``cloud-stack`` utility creates resources in the cloud provider ``aws`` in region ``us-west-2``. To create resources in another cloud provider or region other than the default, use the |ccloud| CLI to view the available cloud providers and regions:
> 

Reviewer: ```suggestion
By default, the ``cloud-stack`` utility creates resources in the cloud provider ``aws`` in region ``us-west-2``. To create resources in another cloud provider or region other than the default, complete the following steps:

#.  View the available cloud providers and regions using the |ccloud| CLI:

```

>    .. code:: bash

Reviewer: ```suggestion
   .. code-block:: bash
```

      ccloud kafka region list

>    Then create the ``ccloud-stack`` and override the parameters ``CLUSTER_CLOUD`` and ``CLUSTER_REGION``, as shown below.

Reviewer: ```suggestion
#. Create the ``ccloud-stack`` and override the parameters ``CLUSTER_CLOUD`` and ``CLUSTER_REGION`` by running the following command:
```

>    .. code:: bash

Reviewer: ```suggestion
   .. code-block:: bash
```

      CLUSTER_CLOUD=aws CLUSTER_REGION=us-west-2 ./ccloud_stack_create.sh

> #. By default, a new `ccloud-stack`` creates a new environment. To reuse an existing environment, create the ``ccloud-stack`` and override the parameter ``ENVIRONMENT`` with an existing environment ID, as shown below.
> 

Reviewer: ```suggestion
By default, a new ``ccloud-stack`` creates a new environment. To reuse an existing environment, create the ``ccloud-stack`` and override the parameter ``ENVIRONMENT`` with an existing environment ID, as shown in the following example:```

>    .. code:: bash

Reviewer: ```suggestion
   .. code-block:: bash
```

      ./ccloud_stack_destroy.sh stack-configs/java-service-account-<SERVICE_ACCOUNT_ID>.config

.. _ccloud-stack-options:

================
Advanced Options
================

Select Cloud Provider and Region
--------------------------------

By default, the ``cloud-stack`` utility creates resources in the cloud provider ``aws`` in region ``us-west-2``. To create resources in another cloud provider or region other than the default, complete the following steps:

#. View the available cloud providers and regions using the |ccloud| CLI:

   .. code-block:: bash

      ccloud kafka region list

> #. Create the ``ccloud-stack`` and override the parameters ``CLUSTER_CLOUD`` and ``CLUSTER_REGION``, as shown below.

Reviewer: ```suggestion
#. Create the ``ccloud-stack`` and override the parameters ``CLUSTER_CLOUD`` and ``CLUSTER_REGION``, as shown in the following example:
```
